db/schema.rb

```diff
@@ -41,4 +41,10 @@
     t.index ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true
   end
 
+  create_table "users", force: :cascade do |t|
+    t.string "email"
```

Comment:
Should we add index on email?

Reply:
It's already generated and you can sort by id of email via Admin Panel